src/main/src/main/java/org/geoserver/util/NoExternalEntityResolver.java

```diff
@@ -25,6 +25,7 @@
  */
 public class NoExternalEntityResolver implements EntityResolver {
 
+    public static final String ERROR_MESSAGE_BASE = "Entity resolution disallowed for ";
```

Comment:
Avoid repeating over and over the same string among differen tests, see all the tests checking for this string